src/Umbraco.Web/Editors/RelationTypeController.cs

```diff
@@ -50,6 +52,23 @@ public RelationTypeDisplay GetById(int id)
             return display;
         }
 
+        public PagedResult<RelationDisplay> GetPagedResults(int id, int pageNumber = 1, int pageSize = 100)
+        {
+
+            if (pageNumber <= 0 || pageSize <= 0)
+            {
+                throw new NotSupportedException("Both pageNumber and pageSize must be greater than zero");
+            }
+
+            // Ordering do we need to pass through?
+            var relations = Services.RelationService.GetPagedByRelationTypeId(id, pageNumber, pageSize, out long totalRecords);
```

Comment:
you need to pass a pageIndex to this service not a 1 based number (see prev comments)

Reply:
I wouldn't worry about ordering for now, people can't sort the results currently.